docs/api.md

```diff
@@ -51,74 +51,33 @@ By using AWS Deep Learning Containers, you can directly run your own training sc
 
 However, for some advanced use cases where you need access to customized tensors from targeted parts of a training script, you can manually construct the hook object. The smdebug library provides hook classes to make this process simple and compatible with the SageMaker ecosystem and Debugger.
 
-#### Hook when using the SageMaker Python SDK
+#### Use smdebug framework hooks to debug SageMaker training jobs
 If you create a SageMaker job and specify the hook configuration in the SageMaker Estimator API
-as described in [AWS Docs](https://docs.aws.amazon.com/sagemaker/latest/dg/train-model.html),
+as described in [AWS Docs](https://docs.aws.amazon.com/sagemaker/latest/dg/train-debugger.html),
 the CreateTrainingJob API operation containing the hook configuration will be automatically written to the training container.
 
-To capture tensors from your training model, paste the following code to the top or the main function of the training script.
+If using the partially supported AWS Deep Learning Containers or custom containers, paste the following code to the top or the main function of the training script to capture tensors from your training model.
```

Comment:
Why do we mention "partially supported" here?

Reply:
I was mentioning the old frameworks that does not have the Debugger hook auto-registration prepared like the ZCC ones.